setup.py

```diff
@@ -2,21 +2,9 @@
 from setuptools import setup, find_packages
 
 
-def find_version(filename):
-    with open(filename) as f:
-        initlines = f.readlines()
-    version_line = None
-    for line in initlines:
-        if line.startswith('__version__'):
-            vstr = line.strip().split()[-1]
-            ver = ast.literal_eval(vstr)
-            break
-    return ver
-
-
 setup(
     name='xpdacq',
-    version=find_version('xpdacq/__init__.py'),
+    version='0.6.0',
     packages=find_packages(),
```

Comment:
Are you reverting?

Reply:
No, what is the current version number? (It will get bumped when we do the next tag)

Reply:
I see. nvm.

0.6.0 is the current version. I thought ``find_version`` will deal with the version automatically.

Reply:
`find_version` is super buggy, @scopatz and I decided that it would be better if we just let rever do its thing on the `setup.py` as well as the `__init__.py`. This PR is designed to go in before the tag.